x-pack/plugins/reporting/public/lib/reporting_api_client/reporting_api_client.ts

```diff
@@ -98,7 +99,7 @@ export class ReportingAPIClient implements IReportingAPI {
    */
   public getReportURL(jobId: string) {
     const downloadLink = this.http.basePath.prepend(
-      `${INTERNAL_ROUTES.JOBS.DOWNLOAD_PREFIX}/${jobId}`
+      `${INTERNAL_ROUTES.JOBS.DOWNLOAD_PREFIX}/${jobId}?${internalAccess}`
```

Comment:
1. Let's use the import from core here:
```suggestion
      `${INTERNAL_ROUTES.JOBS.DOWNLOAD_PREFIX}/${jobId}?${ELASTIC_INTERNAL_ORIGIN_QUERY_PARAM}=true`
```

2. Strangely, it looks like this method (`getReportURL`) is identical to `getDownloadLink` near the end of this file. Would you mind removing the `getDownloadLink` method? Let's just keep `getReportURL`

Reply:
I was having trouble removing getDownloadLink because it's tied to the toast's getUrl method that then is reconciled into getReportURL. Is it ok to keep both to not refactor a bunch of the toast code?